tools/cldr-code/src/main/java/org/unicode/cldr/util/CodePointEscaper.java

```diff
@@ -273,4 +290,54 @@ public static String rawCodePointToEscaped(int codePoint) {
                 ? Integer.toString(codePoint, 16).toUpperCase(Locale.ROOT)
                 : result.toString();
     }
+
+    public static final String getHtmlRows(
+            UnicodeSet escapesToShow, String tableOptions, String cellOptions) {
+        if (!escapesToShow.strings().isEmpty()) {
+            throw new IllegalArgumentException("No strings allowed in the unicode set.");
+        }
```

Comment:
maybe also confirm tableOptions and cellOptions start with spaces if not empty -- either throw IllegalArgumentException or insert the spaces if missing